app/components/preprint-form-section.js

```diff
@@ -42,6 +44,20 @@ export default CpPanelComponent.extend(Analytics, {
      */
     hasOpened: false,
 
+    /**
+     * Does the user have permission to edit information inside this panel?
+     * @property {boolean} canEdit
+     */
+    canEdit: true,
+    /**
+     * Should the preprint-form-block class be added to the form section?
+     * Used for hiding preprint form sections for read contributors
+     * @computed {boolean} addPreprintFormBlock
+     */
+    addPreprintFormBlock: computed('canEdit', 'innerForm', function() {
```

Comment:
This logic was added to help hide form sections on edit, where read contribs can only see one section.  We can prevent the section content from being loaded, but the class on the surrounding div was still formatting the section so it was still partially visible.  This only adds the css formatting if the div content renders.